MIP_institutions.json

```
@@ -1492,6 +1492,30 @@
                 ]
             },
             "consortiums": []
+        },
+        "uoexeter": {
```

```suggestion
        "UoExeter": {
```

Following https://github.com/PCMDI/mip-cmor-tables/issues/60#issuecomment-2328193264